test/cppsuite/test_harness/workload/database_operation.h

```diff
@@ -122,19 +122,18 @@ class database_operation {
 
     /* Basic read operation that walks a cursors across all collections. */
     virtual void
-    read_operation(thread_context &context, WT_SESSION *session)
+    read_operation(thread_context *tc)
```

Comment:
Is there a reason that we're switching to a pointer here? Looks like we're not expecting `nullptr` anyway.

Reply:
Producers a rather hard to solve compiler error